### DISCUSSION TITLE
After latest update, PC waking up from sleep, fancontrol resumes maximized

### DISCUSSION THREAD

**Discussion by fooflighter** [Rem0o/FanControl.Releases#1574] [Q&A]
The minimized button is checked, I've uncheck and rechecked and resaved the config.

Still is full screen or maximized when PC wakes up from sleep

**Comment by Rem0o** [comment]
Try and run updater.exe in FanControl's folder.

**Reply by fooflighter** [reply]
Thanks, I'm running the latest version already

**Reply by Rem0o** [reply]
Run it again, that's the point.

**Reply by fooflighter** [reply]
Thank you, that did the trick
